packages/ember-codemod-remove-ember-css-modules/src/migration/ember-app/steps/shared/update-templates.ts

```diff
@@ -1,30 +1,47 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment, @typescript-eslint/no-non-null-assertion */
 import { readFileSync } from 'node:fs';
 import { join } from 'node:path';
 
 import { ASTHandlebars as AST } from '@codemod-utils/ast';
 import { createFiles } from '@codemod-utils/files';
 
-function sanitizeClassAndLocalClassAttributes(file) {
-  function removeAttributeWithoutValue(attributeName, attributes) {
+import type {
+  Entities,
+  OptionsForUpdateTemplates,
+} from '../../../../types/index.js';
+
+type Data = {
+  __styles__: string;
+};
+
+function sanitizeClassAndLocalClassAttributes(file: string): string {
+  function removeAttributeWithoutValue(
+    attributeName: string,
+    attributes: unknown[],
+  ): void {
```

Comment:
`ember-template-recast@6.1.4` didn't support TypeScript as well as I had hoped. The library [depends on `ASTv1` from `@glimmer/syntax`](https://github.com/ember-template-lint/ember-template-recast/blob/v6.1.4/src/index.ts#L2), while [`@glimmer/syntax` seems to be working on `v2` types](https://github.com/glimmerjs/glimmer-vm/tree/664a746b3baf82221c5874a759279d1afdd41e5b/packages/%40glimmer/syntax/lib/v2).

Rather than installing `@glimmer/syntax` as a development dependency to try to write the correct types, (in the functions that I wrote,) I opted to use `unknown`, ignore TS errors, and rely on tests.